src/replicache.ts

```diff
@@ -272,7 +274,7 @@ export default class Replicache implements ReadTransaction {
       }
     }
 
-    const {batchPushInfo} = syncInfo;
+    const {batchPushInfo} = {...syncInfo};
```

Comment:
we are still not returning this (the syncinfo), you or @arv please file an issue if it is important to prioritize